specs/binding-grpc-kafka.spec/src/test/java/io/aklivity/zilla/specs/binding/grpc/kafka/streams/GrpcProduceIT.java

```diff
@@ -63,6 +63,15 @@ public void shouldExchangeMessageInClientStream() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Specification({
+        "${grpc}/client.stream.rpc.oneway/client",
+        "${grpc}/client.stream.rpc.oneway/server"})
+    public void shouldExchangeMessageInClientStreamOneway() throws Exception
```

Comment:
```suggestion
    public void shouldSendMessagesViaClientStreamOneway() throws Exception
```
Note: exchange implies request-response like echo.